requirements.txt

```
@@ -12,5 +12,5 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-th2-grpc-sim~=5.1.0.dev
-th2-grpc-common~=4.3.0.dev
+th2-grpc-sim~=5.1.0.rc1
```

I believe this is the correct version https://pypi.org/project/th2-grpc-sim/5.1.0rc1/
```suggestion
th2-grpc-sim~=5.1.0rc1
```